### ISSUE TITLE
FEEDBACK

### ISSUE THREAD

**Issue by mike632t** [davidly/ntvcm#3]
David,

I've used this emulator several times in the last couple of weeks and I love it!!   It just works, easy to use, no disk images, no messing about, and the performance stats are surprisingly useful too!

Compiles and runs quite nicely on a Raspberry Pi 3B+ too. 

Thank you!

**Comment by davidly** [comment]
Thanks for writing this! I'm happy to hear it's working. 

Also, it's great to hear it works on the Pi 3B+ -- I haven't tried to build it for a 32-bit platform so I wasn't sure.

David

**Comment by mike632t** [comment]
Actually it is 64 bit - I forgot to mention that I'm running Debian (bookworm).  I can try it on a 32 bit machine tomorrow,

**Comment by mike632t** [comment]
I tried installing Debian 12 on on of my 32-bit machines but ran into GRUB issues that I don't want to fix (as if I do it will break other much older stuff!).

The newest version of Debian I had available on that system was a bit old (Debian wheezy), so I had to compile using:
```
g++ -ggdb -Ofast -fno-builtin -D NDEBUG -I . ntvcm.cxx x80.cxx -o ntvcm -std=c++11
```
However, it compiled without error and seems to work. ``` ntvcm z80doc.com``` completed without error, though I did get some "Line buffer overflow" errors when loading the example BASIC programs.
